src/apps/aave/constants.ts

```diff
@@ -81,3 +81,15 @@ export const AAVE_V3_ADDRESSES_BY_NETWORK_ID: Record<
     pool: '0x07ea79f68b2b3df564d0a34f8e19d9b1e339814b',
   },
 }
+
+export const AAVE_LOGO =
+  'https://raw.githubusercontent.com/valora-inc/dapp-list/main/assets/aave.png'
+
+export const AAVE_POOLS_BASE_URL = 'https://app.aave.com/'
```

Comment:
I was able to get specific networks by clicking on the dropdown. For example https://app.aave.com/?marketName=proto_arbitrum_v3

Reply:
updated in [fc1f27c](https://github.com/valora-inc/hooks/pull/495/commits/fc1f27c081a78f10d92e59746e824c1426320468)